index.html

```diff
@@ -1135,6 +1160,169 @@ <h3>
         </section>
       </section>
     </section>
+    <section>
+      <h2>
+        Application icons and images
+      </h2>
+      <p>
+        Application icons can be used by the user agent in multiple places,
+        such as on a home-screen, in an app store, or even on top of
+        notifications.
+      </p>
+      <p>
+        For this reason, image can be declared with different sizes and for
+        different purposes. A user agent can then use these data to decide
+        which icons to use in different situations. For instance, a user agent
+        may prefer showing icons with the <a data-lt="badge purpose">badge</a>
+        purpose on top of notifications, as <a data-lt=
+        "badge purpose">badge</a> indicates an icon that is can be placed on
+        top of other content, in size constrained situations.
+      </p>
+      <section>
+        <h2>
+          Icon masks and safe zone
+        </h2>
+        <p>
+          Some platforms have their own preferred icon shape, but as web
+          applications should work across multiple platforms, it is possible to
+          indicate that an icon may be supercomposed with a mask, by adding the
+          <a data-lt="masking purpose">masking</a> purpose. This allows the
+          platform to ensure that the icon looks well integrated with the
+          platform, and even apply different masks and background colors in
+          different places throughout the platform.
+        </p>
+        <p>
+          Any icon indicating that it can be supercomposed with a mask, much
```

Comment:
It doesn't feel right to me to define this in the spec. All we should do here is tell developers to seek out guidance from browser vendors. 

Does notification API provide any guidance like this?

Reply:
Not sure, maybe @beverloo knows

Reply:
I am afraid that if we don't add this here, then some vendors will do their own thing and we will end up with a mess